spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Show', type: :feature do
+  before :each do
+    @user = User.create(name: 'Batman', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Superhero',
+                        post_counter: 6)
+    @post = Post.create(title: 'The Dark Night', text: 'This is my first post', comments_counter: 15,
+                        likes_counter: 18, author: @user)
+    @comment1 = Comment.create!(author: @user, post: @post, text: 'This is my first comment')
+    @comment2 = Comment.create!(author: @user, post: @post, text: 'This is my second comment')
+    @comment3 = Comment.create!(author: @user, post: @post, text: 'This is my third comment')
+    @like = Like.create(author: @user, post: @post)
+
+    visit user_post_path(@user, @post.id)
+  end
+
+  describe 'Post show page display' do
+    it 'displays the post title' do
+      expect(page).to have_content('The Dark Night')
+    end
+
+    it 'displays the post author' do
+      expect(page).to have_content('Batman')
+    end
+
```

Comment:
- [Optional] Dear Team, when testing for such values (like an author name) it would be better to bring these values from the database (make it programmatic) instead of a static text. Something like `expect(page).to have_content(@post.author.name)`